animation_nodes/execution/auto_execution.py

```diff
@@ -15,10 +13,6 @@ def executeNodeTrees(nodeTrees):
         nodeTree.autoExecute()
 
 def afterExecution():
-    for scene in bpy.data.scenes:
-        for viewLayer in scene.view_layers:
-            viewLayer.update()
```

Comment:
Do objects that are parented to objects animated by AN still work?

Reply:
Yes, we read from evaluated IDs and write to original IDs.
I will test the parenting.

Reply:
Yes, parents works as expected.